helm-chart-sources/pulsar/templates/admin-console/pulsar-admin-console-ingress.yaml

```
@@ -70,7 +70,8 @@ spec:
   tls:
   - hosts:
     - {{ .Values.pulsarAdminConsole.ingress.host }}
-    secretName: {{ .Values.tlsSecretName }}
+    secretName: "{{ .Values.tls.pulsarAdminConsole.tlsSecretName }}"
```

To ensure backward compatibility, I think we actually want this:

```suggestion
    secretName: {{ .Values.tls.pulsarAdminConsole.tlsSecretName | default .Values.tlsSecretName | quote }}
```